fsimpl/localfs/localfs.go

```diff
@@ -279,3 +279,12 @@ func (l *Local) SetAttr(valid p9.SetAttrMask, attr p9.SetAttr) error {
 	}
 	return nil
 }
+
+// UnlinkAt implements p9.File.UnlinkAt
+func (l *Local) UnlinkAt(name string, flags uint32) error {
+	// Construct the full path
+	fullPath := path.Join(l.path, name)
```

Comment:
On Windows:

```
filetest.go:335: Unlinkat failed: remove C:\Users\RUNNER~1\AppData\Local\Temp\localfs-3104935054/dummy_file.txt: The process cannot access the file because it is being used by another process.
```

Should be filepath.Join instead

Reply:
I went ahead and fixed it, squashed the commits. I realize the other methods have the same problems. A problem for later...